index.css

```diff
@@ -252,6 +269,81 @@
     color: white;
   }
 
+  .popupArticle {
+    position: fixed;
+    top: 50%;
+    transform: translateY(-50%);
+    width: 100%;
+    height: 100%;
+    background-color: #c1c7d065;
+    backdrop-filter: blur(6px);
+    z-index: 30;
+    display: none;
+    align-items: center;
+    justify-content: center;
+  }
+
```

Comment:
- [ ] Good work creating the popup window 💯 . Kindly add the grey line between buttons and skills missing on your popup window
- [ ] Kindly align your buttons in rows as in the FIGMA design

![image](https://user-images.githubusercontent.com/93630700/215908862-ba013b5c-4ade-41e4-bbcf-0611963c75cd.png)

Reply:
Thanks for the great review 😍. I did implement this design, but still will be causing this effect for more readability ✍️ in extremely smaller screens like this one  ( image attached ) , but solved in the other screen sizes ✌️. 
![image](https://user-images.githubusercontent.com/18634639/216028124-2de5944c-2696-442f-ab5b-60fce392741e.png)